HW02/vorobyov_egor/spec/my_array_spec.rb

```diff
@@ -0,0 +1,83 @@
+require('../my_array')
+
+RSpec.describe MyArray do
+  subject { MyArray.new([1, 2, 3, 4, 5]) }
+
+  let(:result_array) { [1, 2, 3, 4, 5] }
+
+  describe '#select' do
+    context 'when passed valid block' do
+      let(:expected_array) { [2, 4] }
+
+      it 'returns an array of trueth values' do
+        expect(subject.select { |el| el.even? }).to eq expected_array
```

Comment:
No need to use `expected` variable for such a short result 
```suggestion
      it 'returns an array of trueth values' do
        expect(subject.select { |el| el.even? }).to eq [2, 4]
```